src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts

```diff
@@ -64,6 +65,24 @@ export class CodeSuggestionWebviewProvider
     if (!found) this.disposePanel();
   }
 
+  async postLearnLessonMessage(suggestion: completeFileSuggestionType): Promise<void> {
+    if (this.panel && suggestion.isSecurityType) {
+      const learnService = new LearnService(suggestion, OpenCommandIssueType.CodeIssue, this.logger);
+      const lesson = await learnService.getLesson();
+      if (lesson) {
+        void this.panel.webview.postMessage({
```

Comment:
Let's catch errors from `webview.postMessage` here and below, and log them. This can help in case something goes wrong in future. It's currently fire and forget, which is little scary 👯